src/main/webapp/basic-search.js

```diff
@@ -29,7 +29,7 @@ import { Location } from './location'
 import { useQuery } from '@apollo/react-hooks'
 import LinearProgress from '@material-ui/core/LinearProgress'
 import gql from 'graphql-tag'
-import { getIn } from 'immutable'
+import { get, getIn, remove } from 'immutable'
```

Comment:
Can you move this up to line 19 with the other imports from immutable? So we don't import the same module on two different lines?

Reply:
good catch, I'll just have vscode clean up the imports